funnel/templates/project.html.jinja2

```diff
@@ -14,7 +14,7 @@
     "@type": "Event",
     "name": "{{ project.title }}",
     "url": "{{ project.url_for(_external=true) }}",
-    "startDate": "{{ project.date }}",
+    "startDate": "{{ project.schedule_start_at }}",
```

Comment:
These need `.isoformat()` as far as I can tell. Check the spec.

Reply:
Using `isoformat()`.